### PR TITLE
Adding Sciprt Dir

### PR DESCRIPTION
**Pull request by ceguiluzrosas**
Following @coryleeio 's advice, I am mimicking the setup from [asym-crypto-yaml](https://github.com/edx/asym-crypto-yaml)

Following this, I will push the new release to PyPi

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,4 +20,9 @@
     ],
     include_package_data=True,
     zip_safe=False,
```
Comment (ceguiluzrosas) [inline setup.py]: Quick Question: should I keep this as false or true? the package is pretty small.

File: zappa_manage/manage.py [hunk 2]
```diff
@@ -10,16 +10,7 @@
     load_private_key_from_string
 )
 
-@click.group()
```
Comment (ceguiluzrosas) [inline zappa_manage/manage.py:13]: All click stuff has been moved to the scripts folder

File: scripts/zappa_manage.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import click
+from pybase64 import b64encode
+
+from asym_crypto_yaml import (
+    load, Encrypted, decrypt_value,
+    load_private_key_from_file,
+    load_private_key_from_string
+)
+
+from zappa_manage import deploy_lambda_envs
```
Comment (ceguiluzrosas) [inline scripts/zappa_manage.py]: There is only 1 click command however I am unsure if whether or not I should import the other functions given how this [blob](https://github.com/edx/asym-crypto-yaml/blob/master/scripts/asym_crypto_yaml.py#L4-L9) in asym_crypto_yaml imports (almost) everything but doesn't use all

Comment (coryleeio) [reply scripts/zappa_manage.py]: Just import what you need IMO

File: scripts/zappa_manage.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+def deploy_lambda_envs(config_file_path, private_key_content, private_key_path, kms_key_arn, lambda_name):
+    """
+    Loads private key to deploy the application's secret values to corresponding lambda
+    :config_file_path = path to config file
+    :private_key_content = content of private key
+    :private_key_path = path to the private key
+    :kms_key_arn = arn for an aws kms_key
+    :lambda_name = name of an aws lambda function
+    """
+    deploy_lambda_envs(config_file_path, private_key_content, private_key_path, kms_key_arn, lambda_name)
```
Comment (coryleeio) [inline scripts/zappa_manage.py]: I think this might accidentally be a recursive call.

File: setup.py (hunk 1 above)
Comment (coryleeio) [reply setup.py]: I think you can set it to true

**Review by coryleeio — CHANGES_REQUESTED** [review]

File: scripts/zappa_manage.py (hunk 3 above)
Comment (ceguiluzrosas) [reply scripts/zappa_manage.py]: Thanks!

File: scripts/zappa_manage.py (hunk 4 above)
Comment (ceguiluzrosas) [reply scripts/zappa_manage.py]: You right! Thanks for catching it. I called the function underneath to `perform_deploy_lambda_env`, updated the import statement above, and made the same change in `manage.py`

**Review by coryleeio — APPROVED** [review]

**Pull request merged**